zipkin-lens/src/components/DiscoverPage/DiscoverPage.jsx

```diff
@@ -11,6 +11,7 @@
  * or implied. See the License for the specific language governing permissions and limitations under
  * the License.
  */
+/* eslint-disable react/no-unescaped-entities */
```

Comment:
We can probably disable this in the config since all our strings will be in i18n blocks in practice.

Reply:
I see!